packages/shared/src/logic/utils.ts

```diff
@@ -98,9 +99,29 @@ export function makePrettyDay(date: Date) {
 }
 
 export function makePrettyShortDate(date: Date) {
-  return format(date, 'MMM dd, yyyy');
+  const day = getDate(date);
+  const suffix = getDaySuffix(day);
+  return format(date, `MMMM do, yyyy`).replace(
+    /\d+(st|nd|rd|th)/,
+    `${day}${suffix}`
+  );
 }
 
+function getDaySuffix(day: number): string {
+  if (day >= 11 && day <= 13) {
+    return 'th';
+  }
+  switch (day % 10) {
+    case 1:
+      return 'st';
+    case 2:
+      return 'nd';
+    case 3:
+      return 'rd';
+    default:
+      return 'th';
+  }
+}
```

Comment:
Why are we using this function? Doesn't `do` give us this? Is there some corner case we need covered that date-fns doesn't cover?

Reply:
Absolutely nothing. Thanks